momento-sdk/build.gradle.kts

```diff
@@ -53,3 +55,53 @@ tasks.named("analyzeIntTestClassesDependencies").configure {
 tasks.named("analyzeTestClassesDependencies").configure {
     enabled = false
 }
+
+fun registerIntegrationTestTask(name: String, testClasses: List<String>) {
+    tasks.register<Test>(name) {
+        description = "Runs $name integration tests"
+        group = "verification"
+
+        val integrationTestTask = tasks.named<Test>("integrationTest").get()
+        classpath = integrationTestTask.classpath
+        testClassesDirs = integrationTestTask.testClassesDirs
+
+        filter {
+            testClasses.forEach { testClass ->
+                includeTestsMatching(testClass)
+            }
+        }
+
+        useJUnitPlatform()
+        testLogging {
+            exceptionFormat = TestExceptionFormat.FULL
+            events("passed", "skipped", "failed")
+        }
+        outputs.upToDateWhen { false }
+    }
+}
+
+registerIntegrationTestTask(
+    "test-auth-service",
+    listOf("momento.sdk.AuthClient*")
+)
+
+registerIntegrationTestTask(
+    "test-cache-service",
+    listOf(
+        "momento.sdk.Cache*",
+        "momento.sdk.DictionaryTest",
+        "momento.sdk.ListTest",
+        "momento.sdk.SetTest",
+        "momento.sdk.SortedSetTest"
+    )
+)
+
+registerIntegrationTestTask(
+    "test-storage-service",
+    listOf("momento.sdk.storage.*")
+)
+
+registerIntegrationTestTask(
+    "test-topics-service",
+    listOf("momento.sdk.Topic*")
+)
```

Comment:
Can we group each service under a package of the same name? That was a change I made in other SDKs to make the pattern matching more maintainable, eg:

- move all auth related tests under a package `momento.sdk.auth`
- cache under `momento.sdk.cache`
- etc